data/adapters/gencode_adapter.py

```diff
@@ -60,41 +67,41 @@ def process_file(self):
             if info['gene_id'].endswith('_PAR_Y'):
                 gene_key = gene_key + '_PAR_Y'
             try:
-                if self.type == 'transcript':
+                if self.label in ['gencode_transcript', 'mm_gencode_transcript']:
                     props = {
                         'transcript_id': info['transcript_id'],
                         'transcript_name': info['transcript_name'],
                         'transcript_type': info['transcript_type'],
                         'chr': data[Gencode.INDEX['chr']],
                         # the gtf file format is [1-based,1-based], needs to convert to BED format [0-based,1-based]
-                        'start:long': str(int(data[Gencode.INDEX['coord_start']]) - 1),
-                        'end:long': data[Gencode.INDEX['coord_end']],
```

Comment:
we need to keep `:long` for numeric values because of the old biocypher compatibility